pkg/networking/client.go

```diff
@@ -0,0 +1,54 @@
+package networking
```

Comment:
I think it should belong to the dcos client (https://github.com/dcos/dcos-cli/blob/master/pkg/dcos/client.go), though for now we don't have an easy way to share it between the 2 projects.

Reply:
Looking at https://docs.mesosphere.com/1.12/api/master-routes/ we have many different api sections in DC/OS, we need to decide which ones would have their own client, and which ones should be under a generic dc/os client.